moveit_planners/trajopt/src/trajopt_planner_manager.cpp

```diff
@@ -33,26 +33,41 @@
  *********************************************************************/
 
 /* Author: Omid Heidari
-   Desc:   Main TrajOpt planning plugin interface to MoveIt
+   Desc:   TrajOpt planning plugin
 */
 
 #include <moveit/planning_interface/planning_interface.h>
-#include <moveit/planning_scene/planning_scene.h>
-#include <moveit/robot_state/conversions.h>
+
+#include <moveit/collision_detection_fcl/collision_detector_allocator_fcl.h>
```

Comment:
Now that you've rebased onto the feature branch, you should be able to use Bullet as a checker. Change this to `collision_dection` (no FCL) and get the loaded Plugin instead (see how OMPL does this)